libs/libpcf/src/base/pcf2place.cpp

```diff
@@ -102,6 +104,19 @@ int pcf2place(const PcfData& pcf_data,
       continue;
     }
 
+    size_t lsb = int_pin.get_lsb();
```

Comment:
Using the LSB as key may not be general purpose, when we consider the custom I/O naming.
See details in the feature:

  https://openfpga.readthedocs.io/en/master/manual/file_formats/io_naming_file/

In such case, the LSB may not be unique. Instead, the pin name should be included. 
My suggestion is to use the coordinate (x, y, z) as the key, which is definitely unique.

Reply:
I have updated code to  use the coordinate (x, y, z) as the key. Please check it, thanks!